### PR TITLE
Add documentation for adding doc string to annotated services

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

File: site/src/sphinx/server-docservice.rst [hunk 1]
```diff
@@ -219,3 +219,21 @@ For gRPC:
...
       </executions>
     </plugin>
+
+For Annotated services:
+
+- Use :api:`@Description` annotation to where you want to describe.
+
+  .. code-block:: java
+
+      import com.linecorp.armeria.server.annotation;
```
Comment (ikhoon) [inline site/src/sphinx/server-docservice.rst]: Maybe?
```suggestion
      import com.linecorp.armeria.server.annotation.*;
```

Comment (minwoox) [reply site/src/sphinx/server-docservice.rst]: Oops Thanks!

File: site/src/sphinx/server-docservice.rst [hunk 2]
```diff
@@ -219,3 +219,21 @@ For gRPC:
         </execution>
       </executions>
     </plugin>
+
+For Annotated services:
+
+- Use :api:`@Description` annotation to where you want to describe.
```
Comment (trustin) [inline site/src/sphinx/server-docservice.rst]: Could be simplified to:
```rst
For annotated service, use :api:`@Description` annotation:
```
...?

File: site/src/sphinx/server-docservice.rst [hunk 3]
```diff
@@ -219,3 +219,21 @@ For gRPC:
...
+
+  .. code-block:: java
+
+      import com.linecorp.armeria.server.annotation.Description;
+
+      @Description("A service that provides user information.")
+      public class UserService {
+
+          @Description("get a user by his or her id")
+          @Get("/users/{id}")
```
Comment (trustin) [inline site/src/sphinx/server-docservice.rst]: - Could swap the two lines for aesthetics' sake? (Longer line appears later.)
- `get...` -> `Retrieves the user information by the given user ID.` ?

File: site/src/sphinx/server-docservice.rst [hunk 4]
```diff
@@ -219,3 +219,21 @@ For gRPC:
...
+          public User getUser(@Param("id") @Description("the id of a user") String id) { ... }
```
Comment (trustin) [inline site/src/sphinx/server-docservice.rst]: - `id` -> `ID`
- Maybe just `the user ID`

File: site/src/sphinx/server-docservice.rst (hunk 2 above)
Comment (minwoox) [reply site/src/sphinx/server-docservice.rst]: Fixed. 😄

**Pull request merged**